### ISSUE TITLE
Openlayers - highlights on hover doesn't work if I group the layers

### ISSUE THREAD

**Issue by andreaordonselli** [tomchadwin/qgis2web#1011]
highlights on hover doesn't work if I group the layers like this in **layers.js**:
	
```
		var layersList = [
		new ol.layer.Group({
					title: '<b>Basemap</b>',
					openInLayerSwitcher: true,		
					layers: [
		lyr_GoogleSatellite,
		]}),
		new ol.layer.Group({
					title: '<b>PRG/PUC/PUG_attuazione</b>',
					openInLayerSwitcher: false,
					layers: [
		lyr_piano_attuativo,
		]}),
		new ol.layer.Group({
					title: '<b>PRG/PUC/PUG_ente</b>',
					openInLayerSwitcher: false,
					layers: [
		lyr_zonizzazione,
		]}),
		new ol.layer.Group({
					title: '<b>Cartografia generale</b>',
					openInLayerSwitcher: true,
					layers: [
		lyr_Idrografia,lyr_ComuniLimitrofi,lyr_Confini,
		]}),
		];
```

to make it work with grouped layers? **qgis2web.js**:  `var onPointerMove = function (evt) {` to make it work with grouped layers?

**Comment by andreaordonselli** [comment]
I found a solution by myself, only this step needs to be changed

```
map.forEachFeatureAtPixel(pixel, function(feature, layer) {
        if (layer) {  ecc.....
```

modify both `var onPointerMove = function (evt) {` and `var onSingleClick = function (evt) {`

**Comment by andreaordonselli** [comment]
I have just released version [3.19.0](https://github.com/qgis2web/qgis2web/releases/tag/v3.19.0), which fixes the issues discussed in this thread. Shortly, the update will be available within the QGIS plugin repository.

Thank you for using the plugin.

I would greatly appreciate it if you would consider donating a virtual coffee through the button on the homepage to support the development of qgis2web.

Best regards.
